### DISCUSSION TITLE
Figma pages are stretched out

### DISCUSSION THREAD

**Discussion by karanmidha** [pocka/figspec#47] [Q&A]
I am using `@figspec/react` version 1.0.3. I am using the `FigspecFileViewer`. I am using it as shown below

`<FigspecFileViewer documentNode={ nodes } renderedImages={ renderedImages } link={ figmaLink } />`

However, I am not getting the desired aspect ratio of my design. The page that I am getting appears to be stretched out.

**_Expected_**
![Actual aspect ratio](https://github.com/pocka/figspec/assets/21328841/7a8371ed-289c-4ea1-b689-e5c15ff967ca)

**_Actual_**
![Wrong aspect ratio](https://github.com/pocka/figspec/assets/21328841/4f9b6c0e-545b-4e8f-8287-ae51c0dac60f)

Is there a way I can fix it? @pocka @yannbf @e111077

**Comment by pocka** [comment]
Are the `renderedImages` stretched out, or does the `<img>` use wrong dimension?
Are guides (red outline on hover and blue outline on select) also stretched out?

**Reply by karanmidha** [reply]
The image is using the wrong dimensions. The rendered image renders correctly (The image in original post with black background comes from clicking the link in `renderedImages`). The inspector appears to be working fine but is not in sync with the image. The inspector is working on the actual coordinates where as the image is stretched out. (Below you can see the inspection of the entire page boundary), that is how the page should appear but it's stretched too wide.


![correct inspect](https://github.com/pocka/figspec/assets/21328841/f02737da-1576-4af8-b86b-552cc27e4f21)

**Reply by pocka** [reply]
I checked the v1 source but found nothing that would affect aspect ratio.
Possible reasons I can think of now are:

- Changes to Figma REST API (less likely)
- Figma REST API parameter things
- Browser bugs/quirks
- Effects (e.g. shadow) or hidden node edge-case in our code

Could you open an issue with a reproduction repo/sandbox (static data is necessary, code is preferred but optional) and affected browser versions?

**Reply by yannbf** [reply]
Is your browser on 100% zoom level? Or did you make it bigger or smaller?
